socketapi.js

```diff
@@ -19,31 +34,63 @@ io.on("connection", (socket) => {
         }
         nextIds[code]++;
         // send board and next id back to user
-        socket.emit("joindata", {nextId: nextIds[code], board: boards[code]})
+        socket.emit("joinData", {nextId: nextIds[code], board: boards[code]})
+    });
+
+    //  drawPen
+    //  Broadcast client drawing with the pen
+    //      data: 
+    //      {
+    //          code: the code for the board being drawn on
+    //          id: the id of the pen object
+    //          size: the stroke-width of the pen object
+    //          color: the color of the pen object
+    //          newPoints: the points being added to the Pen object's path
+    //      }
+    socket.on("drawPen", function(data) {
```

Comment:
This seems like it could be one function since when you add something it will always immediately re-emit regardless of data (but this goes back to the issue/comment I left in BoardEditor.js, I think it should probably be one function)